contracts/storage/OptionsVaultStorage.sol

```diff
@@ -86,6 +86,15 @@ contract OptionsVaultStorageV3 {
     /// @notice Logic contract used to select strike prices
     address public strikeSelection;
 
+    /// @notice Last `round` when strike overriden
+    uint256 public lastStrikeOverride;
+
+    /// @notice Current overriden strike price
+    uint256 public overridenStrikePrice;
```

Comment:
One way to optimize for storage read/writes here is to use a struct to pack these variables together. This can be done because `lastStrikeOverride` is a `uint16` number. See how we do it in [DepositReceipt](https://github.com/ribbon-finance/ribbon-v2/blob/master/contracts/libraries/VaultDeposit.sol#L5)

Reply:
so I am guessing in deposit receipt this works because it crams round (uint16) + amount (uint128) into a uint256 number which does not take as much space as either of them separate. 

so in my case this would only make sense if overridenStrikePrice will fit in uint128 - we can confirm this right

Reply:
Yep that's right, you can  read more about [struct packing here](https://fravoll.github.io/solidity-patterns/tight_variable_packing.html)

For every 256 bits, the EVM allocates a storage slot. As long as we stayed <= 256 we can save on using an extra slot